### PR TITLE
Remote action effects on bitlocker profiles

### PR DESCRIPTION
**Pull request by cgerke**

### CODE REVIEW

File: memdocs/intune/remote-actions/devices-wipe.md [hunk 1]
```diff
@@ -176,6 +176,9 @@ If you want to remove devices from the Intune admin center, you can delete them
 1. Sign in to the [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431).
 2. Choose **Devices** > **All devices** > choose the devices you want to delete > **Delete**.
 
+> [!IMPORTANT]
+> The Delete action will trigger a Bitlocker suspension when Bitlocker encryption is managed by Intune. To create a Bitlocker profile: [Manage BitLocker policy for Windows devices with Intune](../protect/encrypt-devices.md)
```
Comment (JohanFreelancer9) [inline memdocs/intune/remote-actions/devices-wipe.md:180]:
```suggestion
> The delete action will trigger a BitLocker suspension when BitLocker encryption is managed by Intune. To create a BitLocker profile, see [Manage BitLocker policy for Windows devices with Intune](../protect/encrypt-devices.md).
```

File: memdocs/intune/remote-actions/devices-wipe.md [hunk 2]
```diff
@@ -189,6 +192,9 @@ If a removed device checks in before its device certification expires, it will r
 
 The device clean up rule doesn't trigger a wipe or retire.
 
+> [!IMPORTANT]
+> The device clean up rule doesn't trigger a Bitlocker suspension when Bitlocker encryption is managed by Intune. To create a Bitlocker profile: [Manage BitLocker policy for Windows devices with Intune](../protect/encrypt-devices.md)
```
Comment (JohanFreelancer9) [inline memdocs/intune/remote-actions/devices-wipe.md:196]:
```suggestion
> The device clean-up rule doesn't trigger a BitLocker suspension when BitLocker encryption is managed by Intune. To create a BitLocker profile, see [Manage BitLocker policy for Windows devices with Intune](../protect/encrypt-devices.md)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cgerke, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Smritib17 Please review and signoff this PR. Thanks

**Comment by cgerke** [comment]
Any progress on this one?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by cgerke** [comment]
Is this going to get merged?

**Comment by yogkumgit** [comment]
@Smritib17 Gentle reminder. Thanks

**Pull request merged**